docs/content/common-subs/create-ks-core-kind-cluster.md

```diff
@@ -0,0 +1,36 @@
+<!--create-ks-core-kind-cluster-start-->
+create the ks-core kind cluster
+```shell
+export KUBECONFIG=~/.kube/config
+kind create cluster --name ks-core --config - <<EOF
+kind: Cluster
+apiVersion: kind.x-k8s.io/v1alpha4
+nodes:
+- role: control-plane
+  kubeadmConfigPatches:
+  - |
+    kind: InitConfiguration
+    nodeRegistration:
+      kubeletExtraArgs:
+        node-labels: "ingress-ready=true"
+  extraPortMappings:
+  - containerPort: 443
+    hostPort: {{ config.ks_kind_port_num }}
+    protocol: TCP
+EOF
+```
+
+Be sure to apply an ingress control with SSL passthrough to **ks-core**. This is a special requirement for Kind that allows access to the <span class="Space-Bd-BT">KUBESTELLAR</span> core running on **ks-core**.
+```shell
+kubectl apply -f https://raw.githubusercontent.com/kubestellar/kubestellar/main/example/kind-nginx-ingress-with-SSL-passthrough.yaml
+```
+**Wait about 20 seconds** and then check if the ingress control is ready on **ks-core**:
```

Comment:
The second command is also about waiting. It would be simpler to just have that one command. Is 90 seconds not long enough to wait? If we need another 20 seconds then just add 20s to the limit on the `kubectl wait`.